compiler/src/dotty/tools/dotc/transform/CheckUnused.scala

```diff
@@ -500,6 +500,8 @@ object CheckUnused:
     val warnings = ArrayBuilder.make[MessageInfo]
     def warnAt(pos: SrcPos)(msg: UnusedSymbol, origin: String = ""): Unit = warnings.addOne((msg, pos, origin))
     val infos = refInfos
+    //println(infos.defs.mkString("DEFS\n", "\n", "\n---"))
+    //println(infos.refs.mkString("REFS\n", "\n", "\n---"))
```

Comment:
```suggestion
```

Reply:
I could use the local convention
```
// checkConsistent() // DEBUG
```
which appears to have a special code
```
// println(owner.info.decls.toList.map(_.debugString).mkString("\n  ")) // !!! DEBUG
```
or add `config.Printers.lint` but frankly I never edit that config to enable printing. If I want to enable some `typr.println` I just remove `typr`. The many layers of just getting debug output from a test means I don't get debug output.

Reply:
if that's on purpose then it's fine, thanks for explaning!

Reply:
Thanks, normally I have been deleting it, but I've written it a few times now. Uncommented would definitely be accidental; I should add the DEBUG for easy grepping. Or maybe just add the debug printer for lint. I will try that when fully caffeinated.